printmyblog.php

```diff
@@ -1,24 +1,25 @@
 <?php
+
 /**
  * @package PrintMyBlog
  * @version 1.0
+ *
+ * @wordpress-plugin
+ * Plugin Name: Print My Blog
```

Comment:
Oh, yes I suppose those asterixes are fine to add. Go figure!

Reply:
Here you go https://developer.wordpress.org/plugins/plugin-basics/header-requirements/#header-fields